### DISCUSSION TITLE
/zh-cn/blog/posts/marked_with_katex

### DISCUSSION THREAD

**Discussion by giscus** [Reverier-Xu/discussions#138] [Announcements]
# Marked JS 集成 Katex 数学公式渲染 - Reverier's Blog

前言 在内容网站中支持 Markdown 渲染已经是一个很常见的需求了，相比较 Vditor、markdown-it 等重量级 markdown 编辑器与渲染工具来说，用 marked 这类更轻量

https://blog.woooo.tech/posts/marked_with_katex/

**Comment by deltamaya** [comment]
十分有用的文章

**Comment by AkagawaTsurunaki** [comment]
`inlineKatex`函数中，renderer那里应该增加一行
```
options.displayMode = false
```
否则渲染会出问题。因为下面渲染块公式的 `options.displayMode = true` 会覆盖katex的默认渲染行为。
